_includes/head.html

```diff
@@ -0,0 +1,12 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta lang="en">
+  <meta name="generator" content="Jekyll v{{ jekyll.version }}">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/149f23b8d9.js"></script>
+  <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/styles.css">
```

Comment:
I think it should be styles.scss instead of styles.css

Reply:
The Jekyll generates `styles.css` even though definitions are in `scss`
![image](https://user-images.githubusercontent.com/17467768/61795090-e2e09180-ae22-11e9-9400-949e024e8e57.png)

Reply:
Ah ok.